frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -417,10 +423,7 @@ function determineDefaultState(
     ({ position } = urlState);
   }
 
-  let zoomStep = datasetConfiguration.zoom;
-  if (urlState.zoomStep != null) {
-    ({ zoomStep } = urlState);
-  }
+  const { zoomStep } = urlState;
```

Comment:
I'd suggest handling the zoomstep case the same way as the position and rotation, this should fix the bug as well:

```javascript
let zoomStep = datasetConfiguration.zoom;
if (tracing != null) {
  zoomStep = getSomeServerTracing(tracing).zoomLevel;
}
if (urlState.zoomStep != null) {	
  ({ zoomStep } = urlState);	
}
```
(untested)